db/fixtures/lessons/node_js_lessons.rb

```diff
@@ -30,88 +30,109 @@ def node_js_lessons
+      description: 'Controllers in Express',
       is_project: false,
-      github_path: '/nodeJS/express_and_mongoose/express_102_crud_and_mvc.md',
-      identifier_uuid: '2b19f45c-536a-412d-8062-2e5c4144b6b6',
+      github_path: '/nodeJS/express/controllers.md',
+      identifier_uuid: '4c3ee6e9-436d-4d13-b0dc-3a19908efce6',
+    },
+    'Views' => {
+      title: 'Views',
+      description: 'Views in Express',
+      is_project: false,
+      github_path: '/nodeJS/express/views.md',
+      identifier_uuid: '7199ddba-9a78-4786-ae5b-33dbd890f80b',
     },
     'Mini Message Board' => {
       title: 'Mini Message Board',
       description: 'We take a break from the main tutorial and create a simple message board.',
       is_project: true,
-      github_path: '/nodeJS/express_and_mongoose/project_mini_message_board.md',
+      github_path: '/nodeJS/express/project_mini_message_board.md',
       accepts_submission: true,
       previewable: true,
-      identifier_uuid: '0270683a-d1e1-49f1-996e-ab721c251354',
+      identifier_uuid: '3abf7691-690e-43e9-992d-d9fea49d37a4',
     },
-    'Deployment' => {
-      title: 'Deployment',
-      description: "There's nothing quite like seeing your application on a real website. We'll show you how it's done.",
+    'Installing PostgreSQL' => {
+      title: 'Installing PostgreSQL',
+      description: 'How to install PostgreSQL on a local machine.',
       is_project: false,
-      github_path: '/nodeJS/express_and_mongoose/deployment.md',
-      identifier_uuid: 'e62ccde8-866a-47cf-9a41-e56234f08f84',
+      github_path: '/nodeJS/express/installing_postgresql.md',
+      identifier_uuid: '42c192e3-9d49-4029-b76c-4a6a8e714298',
     },
-    'Express 103: Routes and Controllers' => {
-      title: 'Express 103: Routes and Controllers',
-      description: 'We learn how to use the express router and use the MVC pattern to set up controllers for our Library tutorial',
+    'Using PostgreSQL' => {
+      title: 'Using PostgreSQL',
+      description: 'Using PostgreSQL with node-postgres',
       is_project: false,
-      github_path: '/nodeJS/express_and_mongoose/express_103_routes_and_controllers.md',
-      identifier_uuid: '17f1624b-4263-4e25-8aad-69c49e4990e5',
+      github_path: '/nodeJS/express/using_postgresql.md',
+      identifier_uuid: '14563fb8-717d-461d-866f-1806ec965039',
     },
-    'Express 104: View Templates' => {
-      title: 'Express 104: View Templates',
-      description: 'This lesson shows you how to set up view templates.',
+    'Forms and Data Handling' => {
+      title: 'Forms and Data Handling',
+      description: 'How to use forms and its data validation in Express',
       is_project: false,
-      github_path: '/nodeJS/express_and_mongoose/express_104_view_templates.md',
-      identifier_uuid: '3a3cddd5-e2cf-4545-bc3d-60c692018231',
+      github_path: '/nodeJS/express/forms_and_data_handling.md',
+      identifier_uuid: '2db51796-85a1-49d2-a40e-3f94e93b5dd1',
     },
```

Comment:
The lessons are applied in `node_js.rb` as
`Forms and Data Handling` -> `Installing PostgreSQL` -> `Using PostgreSQL`

Even if functionally it makes no difference, probably best to have the lessons in the same order in this file for consistency.